src/v2/Apps/Conversation/Components/Conversations.tsx

```diff
@@ -32,7 +32,7 @@ const Conversations: React.FC<ConversationsProps> = props => {
     .indexOf(selectedConversationID)
 
   return (
-    <Container width={["100%", "375px"]}>
+    <Container width={["100%", "100%", "375px"]}>
```

Comment:
Adjust container width of conversation list to be 100% for small screens too


Before this change:
<img width="816" alt="Screen Shot 2020-07-21 at 12 35 36 PM" src="https://user-images.githubusercontent.com/12748344/88084626-b0ab6200-cb52-11ea-8f2e-5588abdfece6.png">




<br>
<br>

After this change:
<img width="821" alt="Screen Shot 2020-07-21 at 12 35 55 PM" src="https://user-images.githubusercontent.com/12748344/88084643-b4d77f80-cb52-11ea-8de9-f987d2a582b4.png">